The job **`tidy`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/28438475052/job/84270237592) [(plain enhanced)](https://triage.rust-lang.org/gha-logs/rust-lang/rust/84270237592) [(plain)](https://github.com/rust-lang/rust/commit/6fdb77631e5d38a531d74e9b71362788abdc97b4/checks/84270237592/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
 use crate::utils::tests::TestCtx;
 use crate::utils::tests::git::git_test;
-use crate::ChangeId;
 
 pub(crate) fn parse(config: &str) -> Config {
     TestCtx::new().config("check").with_default_toml_config(config).create_config()
Bootstrap failed while executing `test src/tools/tidy tidyselftest --extra-checks=py,cpp,js,spellcheck`
Build completed unsuccessfully in 0:00:47
  local time: Tue Jun 30 10:46:45 UTC 2026
  network time: Tue, 30 Jun 2026 10:46:45 GMT
##[error]Process completed with exit code 1.
```

</details>

--- a/src/bootstrap/src/core/config/tests.rs
+++ b/src/bootstrap/src/core/config/tests.rs
@@ -13,6 +13,7 @@ use super::flags::Flags;
 use super::toml::change_id::ChangeIdWrapper;
 use super::toml::rust::parse_codegen_backends;
 use super::{Config, RUSTC_IF_UNCHANGED_ALLOWED_PATHS};
+use crate::ChangeId;
 use crate::core::build_steps::clippy::{LintConfig, get_clippy_rules_in_order};
 use crate::core::build_steps::llvm::LLVM_INVALIDATION_PATHS;
 use crate::core::build_steps::{llvm, test};
@@ -22,7 +23,6 @@ use crate::core::config::{
 };
 use crate::utils::tests::TestCtx;
 use crate::utils::tests::git::git_test;
-use crate::ChangeId;
 
 pub(crate) fn parse(config: &str) -> Config {
     TestCtx::new().config("check").with_default_toml_config(config).create_config()

